lib/internal/crypto/dsa.js

```diff
@@ -254,6 +254,7 @@ function dsaSignVerify(key, data, algorithm, signature) {
     normalizeHashName(key.algorithm.hash.name),
     undefined,  // Salt-length is not used in DSA
     undefined,  // Padding is not used in DSA
+    kSigEncDER, // TODO(@jasnell): revise the inconsistency with WebCrypto's EC
     signature));
```

Comment:
In the PR I'm preparing the add the `dsaEncoding`, the order of these will need to be switched...
```suggestion
    signature,
    kSigEncDER));
```